main.go

```diff
@@ -73,13 +78,31 @@ func init() {
 	utilruntime.Must(kataconfigurationv1.AddToScheme(scheme))
 
 	utilruntime.Must(peerpodconfig.AddToScheme(scheme))
+
+	utilruntime.Must(peerpod.AddToScheme(scheme))
 	// +kubebuilder:scaffold:scheme
 }
 
 func SetTimeEncoderToRfc3339() zap.Opts {
 	return func(o *zap.Options) {
 		o.TimeEncoder = zapcore.RFC3339TimeEncoder
 	}
+
+}
+
+func setProvider() (cloud.Provider, error) {
+	cloudName := os.Getenv("CLOUD_PROVIDER")
```

Comment:
Is `CLOUD_PROVIDER` env variable available for the operator controller-manager ?

Reply:
no, it assumes it's being set with the CM/Secret which is attached to the pod, now you mentioning it sound like a bad idea, I'll think about it

Reply:
fixed, we fetch CM/Secret at reconcile now